repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -105,7 +106,11 @@ Enables users to write custom filters for Repose using a variety of scripting la
 *IN PROGRESS*
 
 == <<valkyrie-authorization.adoc#,Valkyrie Authorization Filter>>
-*IN PROGRESS*
+This filter performs three distinct functions:
+
+* Per-Device Authorization
+* Response Collection Resource Culling
+* Account Permission to Role Translation
 
 == <<versioning.adoc#,Versioning Filter>>
 *IN PROGRESS*
```

Comment:
Nonono. I just meant that I wrote some of the filters for which the `index.adoc` was not updated. So it was my fault that some of this needed to be done. That's all.